django/core/checks/security/base.py

```diff
@@ -223,3 +245,26 @@ def check_referrer_policy(app_configs, **kwargs):
         if not values <= REFERRER_POLICY_VALUES:
             return [E023]
     return []
+
+
+@register(Tags.security, deploy=True)
+def check_feature_policy(app_configs, **kwargs):
+    if _security_middleware():
+        if settings.SECURE_FEATURE_POLICY is None:
+            return [W024]
+        if (
+            not isinstance(settings.SECURE_FEATURE_POLICY, dict) or
+            not settings.SECURE_FEATURE_POLICY.keys() <= FEATURE_POLICY_DIRECTIVES or
```

Comment:
I wonder if this check is a good idea. On the one hand, it can catch mistakes/misspellings. On the other hand, new values will surely be added in time, and once the programmer adds a value Django doesn't know about, it triggers the warning.

I think the check is valuable, but will be better as a separate warning, so it can be disabled on its own.

Reply:
We've taken this approach for `Referrer-Policy`, but I understand the argument and that this is more likely to change. At this stage I'll keep it as is until I have something more fully fleshed out, but will consider splitting the warning.

As for the spelling issue - that is one of the main reasons. It would be interesting to whether any of the browsers highlight unknown values in the developer console as a warning in which case we could possibly dispense with some of this. In addition, it would be good to attempt a little more validation on the origin - we currently only check whether it starts with `http://` or `https://`...